grading/load_config_json.cpp

```diff
@@ -1813,6 +1813,16 @@ nlohmann::json ValidateANotebook(const nlohmann::json& notebook, const nlohmann:
               assert(value != "");
               assert(description != "");
 
+              auto it_next = it;
+              for (++it_next; it_next != choices.end(); ++it_next){
+
+                nlohmann::json choice_next = it_next.value();
+                std::string value_next = choice_next.value("value", "");
+                if (value == value_next)
+                  std::cout  << "ERROR: " << it.value() << " == " << it_next.value() << std::endl;
```

Comment:
This error message isn't very descriptive.  It would be better to indicate both the problem and a suggested resolution.